package.json

```diff
@@ -20,17 +20,17 @@
     "url": "git://github.com/mcollina/mqtt-level-store.git"
   },
   "dependencies": {
-    "level-browserify": "^1.1.1",
+    "level": "^5.0.0",
     "level-sublevel": "^6.6.1",
     "msgpack5": "^3.5.1"
   },
   "devDependencies": {
     "concat-stream": "^1.6.0",
-    "level-test": "^2.0.3",
-    "levelup": "^1.3.9",
+    "level-test": "^6.0.0",
+    "levelup": "^4.0.1",
     "mocha": "^3.0.0",
-    "mqtt": "^2.0.0",
-    "mqtt-connection": "^3.1.0",
+    "mqtt": "https://github.com/mqttjs/MQTT.js.git",
```

Comment:
I'm not convinced by this change. Why?

Reply:
It seems that there is no new release since v2.18.8. The test https://github.com/mcollina/mqtt-level-store/blob/master/test.js#L99 seems to require some update of MQTT.js that related to publish order after v2.18.8.
If v2.18.9 would be released, I will replace the line.

Reply:
Which PR exactly? I'll issue a backport to the v2 line.

Reply:
|MQTT.js|leveldb library|result|my analysis (including my guess)|
|---|---|---|---|
|2.18.8|level-browerify|success|success by chance|
|2.18.8|level|fail||
|master|level-browerify|success|success as expected|
|master|level|success|success as expected|

I guess that replacing `level-browserify` with `level` causes some timing change.
MQTT.js master doesn't depend on such timing because of timing related fixes, but the fixes isn't reflected to the release yet.

Reply:
Do you know which change caused this? I’ll get 3.0.0 out asap. master is a semver-major.

Reply:
I just checked it. I created minimal cherry picked PR for v2.x.x branch of MQTT.js https://github.com/mqttjs/MQTT.js/pull/950

If you merge it and release as v2.18.9, I remove direct github reference from this PR.

Reply:
I added the comment https://github.com/mqttjs/MQTT.js/pull/950#discussion_r285377377

I think that finding a way to fix error on node `4.3.2` is not valuable. However as you mentioned, removing `4.3.2` requires updating semver.

So I give up v2.18.9 approach. 

> Do you know which change caused this?

This is https://github.com/mqttjs/MQTT.js/pull/950

I will wait 3.0.0.